emojis.json

```diff
@@ -90,7 +90,7 @@
     "category": "people"
   },
   "yum": {
-    "keywords": ["happy", "joy", "tongue", "smile", "face", "silly", "yummy", "nom"],
+    "keywords": ["happy", "joy", "tongue", "smile", "face", "silly", "yummy", "nom", "delicious food", "savouring"],
```

Comment:
Instead of `delicious food` this should just be `delicious` since there's no food.